cypress/integration/upload-download.spec.ts

```diff
@@ -0,0 +1,20 @@
+import {UploadPage} from "../page";
+
+let uploadPage: UploadPage;
+
+describe("Upload File", () => {
+  before(() => {
+    uploadPage = new UploadPage();
+  });
+  it("Should have a Tittle", () => {
+    // Arrange
+    const fileName = "example.json";
```

Comment:
The name of the file should be consulted on the fixtures folder
<img width="724" alt="Screen Shot 2022-07-15 at 8 02 50 AM" src="https://user-images.githubusercontent.com/36927617/179228440-ae194741-89a7-47f5-a7d7-1f560bb78725.png">

Reply:
Hola Manul, precisamente el archivo example.json está almacenado en cypress/fixtures